tools/ci_build/github/azure-pipelines/templates/compliance.yml

```diff
@@ -4,6 +4,11 @@ parameters:
   type: string
   default: x64
 
+- name: vs2022
+  displayName: Visual Studio Version
```

Comment:
Should be "If the Visual Studio version is 2022" or similar.

Reply:
thanks. updated.